### DISCUSSION TITLE
Chrome 如何像 UnoCSS Playground 一样, 查看导出的 __uno.css

### DISCUSSION THREAD

**Discussion by LearningUnity** [unocss/unocss#3397] [Q&A] [answered]
![Snipaste_2023-11-30_10-50-46](https://github.com/unocss/unocss/assets/17720234/704c1803-595d-4dd2-a10d-f3dc1752ac66)

**Comment by zyyv** [comment, marked as answer]
https://unocss.dev/tools/inspector#usage

**Reply by LearningUnity** [reply]
thank you! thank you!! thank you!!!!!!!!!!!!
